flaskr/static/style.css

```diff
@@ -226,3 +226,17 @@ body.dark-mode .content textarea {
   color: #e0e0e0;
   border: 1px solid #555;
 }
+
+.toggle-content {
```

Comment:
**Issue**: The `.toggle-content` class does not have a dark mode variant. Consider adding styles for `.toggle-content` when in dark mode to ensure consistent appearance across themes.

**Fix**: body.dark-mode .toggle-content {
  color: #bb86fc;
}

body.dark-mode .toggle-content:hover {
  color: #9a67ea;
}